onebusaway-android/src/main/java/org/onebusaway/android/nav/FeedbackReceiver.java

```diff
@@ -0,0 +1,135 @@
+package org.onebusaway.android.nav;
+
+import android.app.NotificationManager;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+
+import org.apache.commons.io.FileUtils;
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+
+import java.io.File;
+import java.io.IOException;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.RemoteInput;
+
+import static org.onebusaway.android.nav.NavigationService.LOG_DIRECTORY;
+
+
+public class FeedbackReceiver extends BroadcastReceiver {
+    public static final String TAG = "FeedbackReceiver";
+
+    public static final String TRIP_ID = ".TRIP_ID";
+    public static final String NOTIFICATION_ID = ".NOTIFICATION_ID";
+    public static final String CALLING_ACTION = ".CALLING_ACTION";
+    public static final String ACTION_NUM = ".ACTION_NUM";
+
+    public static final int FEEDBACK_NO = 1;
+    public static final int FEEDBACK_YES = 2;
+
+    public static final int DISMISS_FEEDBACK_NOTIFICATION = 1;
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        Log.d(TAG, "onReceive");
+
+        int notifyId = intent.getIntExtra(NOTIFICATION_ID,
+                NavigationServiceProvider.NOTIFICATION_ID + 1);
+        int actionNum = intent.getIntExtra(ACTION_NUM, 0);
```

Comment:
Instead of checking for an intent extra here, you should be checking for an action:

`String action = intent.getAction();`

Reply:
We modified the code to set the dismiss action as an action of the intent.